noir-projects/noir-protocol-circuits/src/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -292,11 +297,6 @@ impl BaseRollupInputs {
         }
         offset += MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX * 2;
 
-        for j in 0..MAX_NEW_L2_TO_L1_MSGS_PER_TX {
-            calldata_hash_inputs[offset + j] = newL2ToL1msgs[j];
-        }
-        offset += MAX_NEW_L2_TO_L1_MSGS_PER_TX;
-
         let contract_leaf = combined.new_contracts[0];
         calldata_hash_inputs[offset] = contract_leaf.hash();
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 333]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Down here. `CALL_DATA_HASH_LOG_FIELDS = 4`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4726?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->